### PR TITLE
Implemented AJAX form submission to stay on site and present success …

### PR DESCRIPTION
**Pull request by adamrichardturner**
Hi Ellie,

I noticed this form submission could be improved for better UX, so I suggest this AJAX approach.

- Form submission is done via AJAX now and hides the form from the user on successful submission.
- A success message is presented.
- The user isn't redirected anywhere.

Please see your script.js file to make changes to the submission success message, replacing the string in this line:

`status.innerHTML = 'Thanks for your submission!'`

Hope that's OK, tested and working well my end and emails hitting your inbox I hope.

Thanks,

Adam

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,19 +1,56 @@
...
+    body: data,
+    headers: {
+      Accept: 'application/json',
+    },
+  })
+    .then((response) => {
+      if (response.ok) {
+        status.innerHTML = 'Thanks for your submission!'
+        form.reset()
+        formContainer.style.display = 'none' // This hides the form
```
Comment (elliezub) [inline script.js]:
```suggestion
```

File: styles.css [hunk 2]
```diff
@@ -547,7 +547,16 @@ ul {
   margin: 0 20px;
 }
 
+#contact-section-form-wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
```
Comment (elliezub) [inline styles.css]:
```suggestion
```

File: styles.css [hunk 3]
```diff
@@ -547,7 +547,16 @@ ul {
...
 #email-form-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (elliezub) [inline styles.css]:
```suggestion
```

**Review by elliezub — CHANGES_REQUESTED** [review]
I think this should work and put the message below the form 🧐

**Review by adamrichardturner — COMMENTED** [review]
Changes look good and appropriate.

**Comment by elliezub** [comment]
Looks really good, I adjusted it a bit and it's now In the live version!! Thanks so much Adam :) I'll try to convert my other sites forms to this format too!

**Comment by adamrichardturner** [comment]
Oops, looks like I closed it prematurely. You've got everything now though so no problem.

**Pull request closed without merging**